### PR TITLE
[YUNIKORN-1788] Fix UI styling after Angular 16 update

### PR DESCRIPTION
**Pull request by craigcondit**
### What is this PR for?
Fixes the UI glitches that appeared after the Angular 16 update recently.

### What type of PR is it?
* [x] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1788

### Screenshots (if appropriate)
dashboard
queues
apps
nodes

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -247,6 +247,7 @@ ul.navigation-primary {
 
 .help-menu > .menu-trigger {
   background: transparent;
+  font-size: 14px;
```
Comment (wilfred-s) [inline src/app/app.component.scss:250]: This should be the same change as in the .info-card > .menu-trigger` of the dashboard so the **?** and **i** use the same formatting.

**Review by wilfred-s — CHANGES_REQUESTED** [review]
Angular 15 mentions it in the release notes that a tool has to be run it shows slightly different changes than you have made. I did not link that back to the objects we used while migrating. I found it based on your mdc changes:
https://material.angular.io/guide/mdc-migration
We need to run that tool as the base change, apply the healthcheck, about-menu, about-card and help-menu changes on top of that.

File: src/app/app.component.scss (hunk 1 above)
Comment (craigcondit) [reply src/app/app.component.scss:250]: Done

**Comment by craigcondit** [comment]
> Angular 15 mentions it in the release notes that a tool has to be run it shows slightly different changes than you have made. I did not link that back to the objects we used while migrating. I found it based on your mdc changes: https://material.angular.io/guide/mdc-migration We need to run that tool as the base change, apply the healthcheck, about-menu, about-card and help-menu changes on top of that.

Good catch. Didn't realize there was an automated tool for this. I've updated the PR, also fixed a minor issue with the search box on the apps page where height was wrong and size was way too large to fit on a reasonable-sized window.

**Review by wilfred-s — APPROVED** [review]
LGTM

**Pull request closed without merging**